components/CheckoutWizard/Stepper.tsx

```diff
@@ -0,0 +1,27 @@
+import { List } from "@equinor/eds-core-react"
+import { useIntl } from "react-intl"
+import styled from "styled-components"
+
+import { StepperItem } from "./StepperItem"
+
+const StyledStepper = styled(List)`
+  display: grid;
+  grid-auto-flow: column;
+  grid-auto-columns: 1fr 1fr min-content;
+  padding: 0;
+`
+
+type Props = {
+  currentStep: number,
+}
+
+export const Stepper = ({ currentStep }: Props) => {
```

Comment:
i'd prefer we stick to using `FunctionComponent` as it sets the return type of the function as well